cvcl/forms.py

```diff
@@ -42,18 +43,20 @@ def clean_data_file(self):
 
         return f
 
-    def process_data(self):
+    def process_data(self, course_id):
         f = io.TextIOWrapper(self.cleaned_data['data_file'].file)
         reader = csv.DictReader(f)
 
         for user in reader:
             email = user['Username'] + "@drexel.edu"
```

Comment:
Do not hardcode domain, it should be configurable in settings.py. Otherwise, it's useless outside Drexel.